execution/client/ethclient/engine.go

```diff
@@ -138,21 +138,19 @@ func (s *Client) GetPayload(
 
 // GetPayloadV3 calls the engine_getPayloadV3 method via JSON-RPC.
 func (s *Client) GetPayloadV3(
-	ctx context.Context, payloadID engineprimitives.PayloadID,
+	ctx context.Context,
+	payloadID engineprimitives.PayloadID,
 ) (ctypes.BuiltExecutionPayloadEnv, error) {
 	var t *ctypes.ExecutionPayload
 	result := &ctypes.ExecutionPayloadEnvelope[*engineprimitives.BlobsBundleV1[
-		eip4844.KZGCommitment, eip4844.KZGProof, eip4844.Blob,
+		eip4844.KZGCommitment,
+		eip4844.KZGProof,
+		eip4844.Blob,
 	]]{
 		ExecutionPayload: t.Empty(version.Deneb),
```

Comment:
This is key point of this cleanup: ExecutionPayload is never nil, cause Empty is the default empty `ExecutionPayloadEnvelope` struct.
I added a UT to verify it